exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -22,11 +22,17 @@ function remainderOf(num) {
     If you're not sure, step out of the code and use pen + paper. Start
     with simple examples and pay attention to the process you carry out.
   */
+ return num % 3
```

Comment:
This function should have the **remainderOf(n, d)** signature, and return the remainder of n/d, instead of n/3.